src/server/database/repositories/client/service.ts

```diff
@@ -42,6 +42,39 @@ function createPreparedStatement(db: DBType) {
         },
       })
       .prepare(),
+    findAllPublicFiltered: db.query.client
+      .findMany({
+        where: or(
+          like(client.name, sql.placeholder('filter')),
+          like(client.ipv4Address, sql.placeholder('filter')),
+          like(client.ipv6Address, sql.placeholder('filter'))
```

Comment:
use `ilike` for case insensitive search. and remove `toLowerCase()` from the inputs

Reply:
yes my bad. then client.name should also be lower, otherwise the casing would break the search